### PR TITLE
Updates to be compatible with React 16

### PR DESCRIPTION
**Pull request by expobrain**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,7 +100,7 @@
     "highlight.js": "^9.3.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": ">=0.14.0 || <=16.0.0",
+    "react-dom": ">=0.14.0 || <=16.0.0"
```
Comment (bvaughn) [inline package.json:104]: We're getting ready to release react@16.1.0. (We released a beta just last week.) I don't want to update versions only to lock people in at 16.0.0. I think a better version patter would be: `^0.14.0 || ^15.0.0 || ^16.0.0`

**Review by bvaughn — COMMENTED** [review]
This PR has conflicts with `master` now after I merged your other PR. I'll just make this change myself in the interest of time.

**Comment by bvaughn** [comment]
v1.0.6 just released with 16 compatibility. Thanks for the nudge.

File: package.json (hunk 1 above)
Comment (expobrain) [reply package.json:104]: Oh, crap, you are right, make sense. Thank you!

Comment (bvaughn) [reply package.json:104]: No worries 😄

**Pull request closed without merging**